NEWS.md

```diff
@@ -8,6 +8,8 @@
 ## Minor improvements and bug fixes
 
 * Pass through warnings when `options(warn = 2)` is set (#721, @yutannihilation).
+* Progress reporter now generates a context from the filename and no longer
+  errors if tests occur before a context (#700, #705).
```

Comment:
Other reporters are OK without a context?

Reply:
Yeah other than the JUnit one the rest seem ok